ccd-definition/CaseField.json

```diff
@@ -752,5 +752,13 @@
     "Label": "Defendant response date",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "localAuthorityPolicy",
```

Comment:
guess local authority doesn't make sense for us. These will need to be named defendant / claimant + 1 / 2 + organisationPolicy

Reply:
And where do we populate these 4 fields? It's not quite clear to me how these fields are going to be used 🤔

Reply:
I think these will all exist in the create a claim event. Unsure how design will implement a this case has 2 defendants etc, but in my mind they will optionally show based on some information entered earlier. With the 4 separate fields we can hard code the case roles for each org policy, so that when I invite an org through the UI as a user, any solicitors assigned via the caa will be given the correct role

Reply:
How do we get organisations ids in create claim event if we search it using auth token?

Reply:
The org policy field is actually the org search used for respondent journey. When the event is started xui makes a call to prd fetching ALL orgs. The user will search and pick one, adding the id.

Reply:
I guess FPLs use case is slightly different, as I don't think they are implementing the respondent journey stuff quite yet. As they are onboarding local authorities they will want to add the id of the org the user belongs to probably in a callback at case creation stage right? We will not need to do this, because when mixed with respondent journey, the user will be entering this information for us.

Reply:
that makes much more sense now, thanks 👍 updated the PR